src/components/get-involved-components/GetInvolvedCard.js

```diff
@@ -9,14 +9,22 @@ const GetInvolvedCard = ({ data }) => {
   const buttonText = data?.linkTitle;
   const link = data?.link;
 
+  const overlayStyle = {
+    backgroundSize: '100% 600%',
```

Comment:
can we experiment with a smaller background size for the hover? I think it might be cool if the gradient didn't span the whole component 👍

Reply:
done. let me know if you want any edits. making the gradient not span the whole thing felt a little abrupt. so i just lessened it
![image](https://github.com/user-attachments/assets/f5e77746-ff0d-4fa3-aa0c-0d6777750b6b)